nomad/plan_endpoint_test.go

```diff
@@ -49,3 +50,80 @@ func TestPlanEndpoint_Submit(t *testing.T) {
 		t.Fatalf("missing result")
 	}
 }
+
+// TestPlanSubmitEndpoint_Bad asserts that the Plan.Submit endpoint rejects bad data
+// with an error instead of panicking.
+func TestPlanEndpoint_Submit_Bad(t *testing.T) {
```

Comment:
I like the name, strong and to the point.

Reply:
Ha, you made me notice I typo'd it in the comment. Thanks :grin: